spec/features/trainer_spec.rb

```diff
@@ -281,6 +291,7 @@
       card = user.cards.find_by(block_id: block.id)
       card.update_attribute(:review_date, Time.now - 3.days)
       visit trainer_path
+      click_link 'ru'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.